### ISSUE TITLE
Extension search/VSCode update fail with XHR

### ISSUE THREAD

**Issue by geauxtigers** [microsoft/vscode#231194]
Type: <b>Bug</b>

Start VSC, select extensions, and attempt to search

VS Code version: Code 1.94.2 (384ff7382de624fb94dbaf6da11977bba1ecd427, 2024-10-09T16:08:44.566Z)
OS version: Windows_NT x64 10.0.19045
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2808)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_graphite: disabled_off<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled<br>webnn: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.83GB (6.44GB free)|
|Process Argv|--crash-reporter-id ae69abb5-4aca-4c3f-a2fa-985d71d21800|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (3)</summary>

Extension|Author (truncated)|Version
---|---|---
debugpy|ms-|2024.6.0
python|ms-|2024.8.1
vscode-pylance|ms-|2024.6.1

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805:30301674
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:31024239
c4g48928:30535728
azure-dev_surveyone:30548225
a9j8j154:30646983
962ge761:30959799
pythongtdpath:30769146
welcomedialog:30910333
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pythonmypyd1:30879173
h48ei257:31000450
pythontbext0:30879054
accentitlementst:30995554
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
cppperfnew:31000557
dsvsc020:30976470
pythonait:31006305
jchc7451:31067544
chatpanelt:31048053
dsvsc021:30996838
945dj816:31013170
pythoncenvpt:31062603
a69g1124:31058053
dvdeprecation:31068756
pythonprt:31056678
dwnewjupytercf:31046870
legacy_priority:31077528

```

</details>

I have had a similar issue in the past https://github.com/microsoft/vscode/issues/173464  

I am having trouble accessing VSCode marketplace through extensions within VSCode.  It has been a problem for quite some time.  This issue has come and gone.  The last time VSCode updated was in June and, I assume, no extensions have updated since then.  Numerous solutions have been recommended and tried:

-	Checking, uninstalling, deactivating antivirus and firewall.
-	Reviewing proxy settings
-	Uninstalling and re-installing VSCode
-	Running the Insiders Edition of VSCode
-	Other actions that I can’t even recall.

None have worked long term.  Once, I was also having a problem reaching github.com in the browser but that has since resolved.  I am not sure if the two are related.  I can access github at the moment.  I manually downloaded the current version of VSC and am currently running the latest.
[vscode-app-1728754946852.log](https://github.com/user-attachments/files/17351084/vscode-app-1728754946852.log)

I have attached the output from developer tools console.  I start VSCode, open dev tools console and select the extensions icon.  I then search for an extension.  This is the output.

**Comment by sandy081** [comment]
Seeing following error - Failed to load resource: net::ERR_CONNECTION_RESET

Seems network related

**Comment by deepak1556** [comment]
If you still experience the issue with latest stable, can you start the application with `--log-net-log=<replace-path>/netlog.json` perform the failing network requests and attach the log in this issue.

**Comment by geauxtigers** [comment]
Yes, the issue still persists.  Here is the .json file.[netlog.json](https://github.com/user-attachments/files/18141559/netlog.json)

**Comment by geauxtigers** [comment]
I have no other information to add as the problem persists.  Thanks for trying to help!
